src/groups/mqb/mqbblp/mqbblp_remotequeue.cpp

```diff
@@ -483,6 +483,19 @@ RemoteQueue::RemoteQueue(QueueState*       state,
     os << '@' << d_state_p->uri().asString();
     d_state_p->setDescription(os.str());
 
+    if (deduplicationTimeMs <= 0) {
+        d_pendingPutsTimeoutNs = 5 *
+                                 bdlt::TimeUnitRatio::k_NANOSECONDS_PER_MINUTE;
+        BALL_LOG_WARN << "Remote queue [" << d_state_p->description()
+                      << "]: cannot schedule PUT deduplication timer with a "
+                      << "non-positive timeout from config ["
+                      << deduplicationTimeMs << " ms], use a default PUT "
+                      << "deduplication timeout for scheduler instead ["
+                      << bmqu::PrintUtil::prettyTimeInterval(
```

Comment:
let's print the minutes, easier to read?

Reply:
Pretty time interval prints `5.00 m`, you want to print the full `5 minutes` instead, right?

Reply:
If this is `5.00 m`, then it is ok.  Just did not want long string of zeroes